git-gutter.el

```diff
@@ -245,8 +254,11 @@ gutter information of other windows."
 
 (defsubst git-gutter:in-repository-p ()
   (cl-loop for vcs in git-gutter:handled-backends
```

Comment:
Now when this function is called, it sets the buffer-local variable `git-gutter:repo-base` to the result of `git-gutter:vcs-check-function`. `git-gutter:repo-base` now contains the root directory tracked by version control.